tests/units/states/test_state_base.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from abc import ABCMeta
+
+from urnai.states.state_base import StateBase
+
+
+class TestStateBase(unittest.TestCase):
+
+    def test_abstract_methods(self):
+        StateBase.__abstractmethods__ = set()
+
+        class FakeState(StateBase):
+            def __init__(self):
+                super().__init__()
+
+        f = FakeState()
```

Comment:
Better would be `fake_state` instead of just `f`. This was in the last PR (and will be fixed in the future), so we should be careful not to repeat it.